coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -197,3 +197,22 @@ def from_metadata(cls, doccomment, docstyle_definition,
 
         return DocumentationComment(assembled_doc, docstyle_definition, indent,
                                     marker, range)
+
+    def assemble(self):
+        """
+        Assembles parsed documentation to the original documentation.
+
+        This function assembles the whole documentation comment, with the
+        given markers and indentation.
+        """
+        lines = self.documentation.splitlines(keepends=True)
+        assembled = self.indent + self.marker[0]
+        if len(lines) == 1:
+            assembled += lines[0]
+        else:
+            assembled += ''.join('\n' if line == '\n' and not self.marker[1]
+                                 else self.indent + self.marker[1] + line
+                                 for line in lines)
```

Comment:
got a corner case where it doesn't work:
Assume this doccomment (with markers `'/**', ' *', '*/`):
```
    /** line 1
     *  line2 */
```
the documentation string would be:
```
' line1\n  line2'
```
gets assembled to
```
    /**     * line1
     *  line2*/
```
please fix that + add a test case for that :+1: